centos7-minimal.cfg

```diff
@@ -0,0 +1,51 @@
+#  Author: Tomasz Filipiec <tomasz.filipiec@gmail.com>
+#   Desc.: Simple kickstart file for automated installation
+#      OS: CentOS 7 Minimal (core)
+# Version: 1.0
+
+# Turning on text-mode installation (little quicker than GUI)
+text
+
+# Setting up authentication and keyboard
+auth --enableshadow --passalgo=sha512
+keyboard --vckeymap=us --xlayouts='us'
+
+# Installation files source (CentOS-7.0-1406-x86_64-Minimal.iso)
+cdrom
+
+# Using only primary disk, ignoring others
+ignoredisk --only-use=sda
+
+# Setting up language to English
+lang en-US.UTF-8
+
+# Setting up network interface to DHCP
+network --bootproto=dhcp --ipv6=auto --hostname=centos-ks.local --activate
+
+# Root password (remember that plaintext only for information purposes)
+rootpw --plaintext centos
+
+# Setting up firewall and enabling SSH for remote management
+firewall --enabled --service=ssh
+
+# Setting timezone
+timezone Europe/Warsaw --isUtc
+
+# Setting up Security-Enhanced Linux into enforcing
+selinux --enforcing
+
+# Setting up MBR
+bootloader --location=mbr --boot-drive=sda
+
+# Setting up Logical Volume Manager and autopartitioning
+clearpart --all --drives=sda --initlabel
+autopart --type=lvm
+
+# Eject cdrom and reboot
+reboot --eject
+
+# Installing only packages for minimal install 
+%packages
+@Core
```

Comment:
Hi @Frodox 
According to official Red Hat documentation for Kickstart - no, first uppercase :-)

https://access.redhat.com/documentation/en-US/Red_Hat_Enterprise_Linux/7/html/Installation_Guide/sect-kickstart-syntax.html#sect-kickstart-packages